src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -88,27 +131,22 @@ protected static ConfigDef baseConfigDef() {
       int order = 0;
       configDef
           .define(TYPE_NAME_CONFIG, Type.STRING, Importance.HIGH,
-                  "The Elasticsearch type name to use when indexing.",
+                  TYPE_NAME_DOC,
                   group, ++order, Width.SHORT, "Type Name")
```

Comment:
This is specific to configs and the calls to `define`. I have to admit, for methods that take 4+ arguments it's much easier for me to figure out whether it's the version that accepts a default or not for instance, when it's one arg per line. I wasn't the first to use 1-arg per line but I think is good to catch mismatches/missing arguments. Finally it's easier to reformat with Alt-Cmd-L when intellij is used. In any case, don't feel too strong about it, as with most stylistic comments.